### DISCUSSION TITLE
Is there development roadmap or plan?

### DISCUSSION THREAD

**Discussion by ArtemLiA** [etna-team/etna#444] [General]
Hi! I want to ask if there is a roadmap or plan for further development of ETNA? If there is no such roadmap, then I would like to suggest creating one. Such roadmap could contain the main directions of ETNA development at the moment (for example expanding functionality, optimization, etc.), This will allow contributors and developers to propose improvements and fixes revelant at the moment.

Thank you!

**Comment by d-a-bunin** [comment]
We are planning our work once in a quarter, and start some tracks to work with. These tracks usually have an active milestone and developers working on them. Tracks are started depending on:
- demands for solving our other projects
- our current understanding of things we should improve

The second step could be improved by getting more user feedback. So, we opened ["Improvements" section in discussions](https://github.com/etna-team/etna/discussions/categories/improvements) to allow users to suggest their vision about ways to improve the library and vote for the improvements they consider important. Unfortunately, there isn't much activity there, but it could be due to the fact that it opened not a long time ago.

For external contributors we recently updated [contribution guide](https://github.com/etna-team/etna/blob/master/CONTRIBUTING.md) to explain how to choose tasks to work with, there are plenty of possible tasks which don't belong to any track right now.

**Comment by ArtemLiA** [comment]
This is, do I understand correctly that before starting to implement new improvements, your team publishes all changes in discussions/issues on github?

**Reply by d-a-bunin** [reply]
The discussion section is fairly new for our repo, so we haven't used it that much for in-team discussions. But we started moving our own ideas about improvements into this place.
For the tasks we usually make issues, but discussion of the specification usually happens in our in-team channels.
